### DISCUSSION TITLE
Vector3 값에 곱하기나 더하기 연산이 안되나요?

### DISCUSSION THREAD

**Discussion by ssm00** [naverz/zepeto-studio-kor#692] [5. Q&A]
백터값에 스피드를 곱하려고 하는데 에러가 나네요
Vector3.right * 2
이런 연산은 불가능한건가요??

**Comment by mayko1234** [comment]
이거 참고해보셔도 안되나요?
[link](https://github.com/naverz/zepeto-studio-kor/discussions/481)

**Reply by ssm00** [reply]
감사합니다 어제 찾아본다고 찾아봤는데 왜 못봤을까요 ㅠㅠ
